### PR TITLE
upcoming: [M3-8754] - Add Alias Tokens to the Theme

### PR DESCRIPTION
**Pull request by harsh-akamai**
## Description 📝
Alias tokens should be available to components via `theme.*`

## Changes  🔄
- Added alias tokens:
  - `accentTokens`
  - `actionTokens`
  - `backgroundTokens`
  - `borderTokens`
  - `contentTokens`
  - `elevationTokens`
  - `interactionTokens` 
  - `typographyTokens`
  - `radiusTokens`
  
## Target release date 🗓️
10/24

### Verification steps
- Ensure everything builds properly

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/ui/src/foundations/themes/index.ts [hunk 1]
```diff
@@ -5,10 +5,26 @@ import { darkTheme } from './dark';
...
+  AccentTypes as AccentTypesLight,
+  ActionTypes as ActionTypesLight,
+  BackgroundTypes as BackgroundTypesLight,
+  BorderTypes as BorderTypesLight,
   ChartTypes,
+  ContentTypes as ContentTypesLight,
+  ElevationTypes as ElevationTypesLight,
   InteractionTypes as InteractionTypesLight,
+  TypographyTypes,
+  RadiusTypes,
```
Comment (harsh-akamai) [inline packages/ui/src/foundations/themes/index.ts:17]: `Typography` and `Radius` are the same for light/dark so simplified our Theme/ThemeOptions for now

**Review by bnussman-akamai — COMMENTED** [review]
Rather than having things like `theme.borderTokens`, `theme.chartTokens`, I wonder if it would make sense to do something like `theme.tokens.border`, `theme.tokens.chart` @jaalah @harsh-akamai

**Comment by harsh-akamai** [comment]
@bnussman-akamai This is a good suggestion. Though it would require a change in the ui package ([breakpoints.ts](https://github.com/linode/manager/blob/develop/packages/ui/src/foundations/breakpoints.ts#L16))

I believe I will have to create a separate PR for a change in the ui package?

**Comment by mjac0bs** [comment]
I also think we're going to want to be consistent with naming conventions. The following PRs  (cc @pmakode-akamai , @zaenab-akamai) are using the previous method of naming with `Tokens`:

- https://github.com/linode/manager/pull/11171
- https://github.com/linode/manager/pull/11165
- https://github.com/linode/manager/pull/11169

@jaalah-akamai Do we want to proceed across the board with `<alias>Tokens` or use `tokens.<alias>` as Banks previously suggested?

**Comment by jaalah-akamai** [comment]
@harsh-akamai @bnussman-akamai @mjac0bs Let's do `theme.tokens.*` - I like that a lot

**Comment by harsh-akamai** [comment]
> The interaction.Background.Secondary uses a darker shade of black in dark mode.

@mjac0bs  Looks like there is no perfect replacement for `bgAccessRow` amongst the existing design tokens. We might have to bring back `bgAccessRow`

cc @jaalah-akamai

**Review by bnussman-akamai — APPROVED** [review]

**Review by jaalah-akamai — APPROVED** [review]

**Review by mjac0bs — APPROVED** [review]

**Pull request merged**